### PR TITLE
Syncs develop and master and bumps to develop version 1.1.2dev

### PR DESCRIPTION
**Pull request by kbwestfall**
As titled.

### CODE REVIEW

**Review by profxj — APPROVED** [review]
Approved

**Comment by kbwestfall** [comment]
Closing because I need to tag 1.1.1 and this would have to be redone again anyway.

File: CHANGES.rst [hunk 1]
```diff
@@ -23,44 +32,42 @@
...
+- Coadd2D offsets
+- Some Coadd2D docs
+- Manual extraction
+- Improve LBT/LUCI
+- Add MMT/MMIRS
+- QL script for Keck/MOSFIRE (beta version)
+- Correct det bug in keck_lris
+- Modifications to allow for flailing LRISr detector
+- Modifications for parse LRIS LAMPS prior to 2010 upgrade
+- Add P200/TripleSpec
```
[the remark this answers is not in the document]
Comment (kbwestfall) [reply CHANGES.rst:61]: Good catch.  Added.

File: doc/step-by-step.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-====================
+=_===================
```
[the remark this answers is not in the document]
Comment (kbwestfall) [reply doc/step-by-step.rst:1]: Another good catch.  Fixed.

**Review by finagle29 — APPROVED** [review]
Looks good to me!

**Pull request merged**